development/prow-addons-ctrl-manager/config/deploy/deploy.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: v1
```

Comment:
this file is generated by kustomize, I do not see an option in kustomize to add own comments at the top of the file. Because this file is autogenerated I also do what to add manually comments here. Let's discuss that